packages/embed/src/factories/create-widget/create-widget.ts

```diff
@@ -66,22 +75,31 @@ export const createWidget = (formId: string, options: WidgetOptions): Widget =>
   }
 
   if (widgetOptions.enableFullscreen) {
+    let backgroundColor = ''
     const { container } = options
     const autoResize = makeAutoResize(container)
     const originalHeight = container.style.height
     const openPopup = () => {
       container.classList.add('tf-v1-widget-fullscreen')
+      if (options.opacity !== undefined) {
+        container.style.backgroundColor = backgroundColor
+      }
       autoResize()
       window.addEventListener('resize', autoResize)
     }
+    const onTheme = (data: any) => {
```

Comment:
you might want to use `unknown` instead of `any`

Reply:
Then I get `TS2571: Object is of type 'unknown'.` error on next line. How would I solve that?

Reply:
I dont see any benefit of `unknown` over `any` - you dont know the type either way. I'd rather type it properly- there are multiple callback methods like this in the code, I will create a technical ticket for this.